Resources/Locale/en-US/chat/managers/chat-language.ftl

```diff
@@ -0,0 +1,15 @@
+chat-language-Universal-name = Universal
+chat-language-Bubblish-name = Bubblish
+chat-language-RootSpeak-name = RootSpeak
```

Comment:
Is it `RootSpeak` or `Rootspeak`?
https://github.com/Simple-Station/Einstein-Engines/blob/master/Resources/Locale/en-US/language/languages.ftl uses `Rootspeak`